v2.1/admin-ui-statements-page.md

```diff
@@ -0,0 +1,110 @@
+
+To view additional details of a statement fingerprint, click on the statement fingerprint in the **Statement** column to see the **Statement Details** page.
+
+## Statement Details page
+
+The **Statement Details** page displays the details of the execution count, latency by phase, row count, and statistics for the selected statement fingerprint.
+
+### Execution count
+
+The **Execution Count** table provides information about the following parameters in numerical values as well as bar graphs:
+
+Parameter | Description
+-----|------------
+First Attempts | The cumulative number of first attempts to execute the SQL statement (or multiple statements having the same fingerprint).
+Retries | The cumulative number of retries to execute the SQL statement (or multiple statements having the same fingerprint).
+Max Retries |
+Total |
+
+### Latency by phase
+
+The **Latency by Phase** table provides the mean and standard deviation values of the service latency for each execution phase (parse, plan, run, and overhead) for the SQL statement (or multiple statements having the same fingerprint). The table provides the service latency details in numerical values as well as bar graphs. The bar graphs are color-coded per execution phase:
+
+Phase | Color code
+-----|------------
+Parse | Red
+Plan | Yellow
+Run | Blue
+Overhead | Red
+
+### Row count
+
+The **Row Count** table provides the mean and standard deviation values of cumulative count of rows returned or affected by the SQL statement (or multiple statements having the same fingerprint). The table provides the service latency details in numerical values as well as a bar graph.
+
+### Statistics
+
+The statistics box on the right-hand side of the **Statements Details** page provides the following details for the statement fingerprint:
+
+Parameter | Description
+-----|------------
+Total time | The cumulative time taken to execute the SQL statement (or multiple statements having the same fingerprint).
+Execution count | The total number of times the SQL statement (or multiple statements having the same fingerprint) is executed.
+Executed without retry | The total number of times the SQL statement (or multiple statements having the same fingerprint) is executed successfully on the first attempt.
+Mean service latency | The average service latency of the SQL statement (or multiple statements having the same fingerprint).
+Mean number of rows | The average number of rows returned or affected while executing the SQL statement (or multiple statements having the same fingerprint).
+
```

Comment:
Are you planning on adding a section on the little table below that statistics box?

Reply:
![image](https://user-images.githubusercontent.com/21692034/42045113-b5b8f492-7ac8-11e8-89e7-4032de46ad62.png)